std/algorithm/iteration.d

```diff
@@ -788,24 +801,6 @@ Params:
     pred = predicate to apply to each element of the range
     r = range or iterable over which each iterates
 
-Example:
----
-void deleteOldBackups()
-{
-    import std.algorithm, std.datetime, std.file;
-    auto cutoff = Clock.currTime() - 7.days;
-    dirEntries("", "*~", SpanMode.depth)
-        .filter!(de => de.timeLastModified < cutoff)
-        .each!remove();
-}
----
-
-If the range supports it, the value can be mutated in place. Examples:
----
-arr.each!((ref a) => a++);
-arr.each!"a++";
----
-
 If no predicate is specified, $(D each) will default to doing nothing
```

Comment:
This paragraph is now attached to the description of `r` under `Params`. It doesn't seem to fit there. Maybe it should be moved before the `Params` section?